playbooks/roles/l2tp-ipsec/templates/instructions-fr.md.j2

```diff
@@ -0,0 +1,127 @@
+- - -
+{% for key, value in streisand_languages.iteritems() %}
+  [{{ value.language_name }}](index{{ value.file_suffix }}.html)&nbsp;
+{% endfor %}
```

Comment:
That's a good suggestion; I think most can be, but there are some exceptions such as `streisand-gateway/templates/instructions*.md.j2`, where the filename doesn't start with `index`. I suspect those will be exceptions to the rule, unless a better approach is presented.